drracket-tool-lib/drracket/private/standalone-module-browser.rkt

```diff
@@ -1056,54 +1044,53 @@
                                                           v))))))
           
-          (let ([levels (sort (hash-map level-ht list)
-                              (λ (x y) (<= (car x) (car y))))])
-            (let loop ([levels levels]
-                       [major-dim 0])
-              (cond
-                [(null? levels) (void)]
-                [else
-                 (let* ([level (car levels)]
-                        [n (car level)]
-                        [this-level-snips (cadr level)]
-                        [this-minor (apply + (map (if vertical? 
-                                                      (λ (x) (get-snip-width x))
-                                                      (λ (x) (get-snip-height x)))
-                                                  this-level-snips))]
-                        [this-major (apply max 0
-                                           (map (if vertical?
-                                                    (λ (x) (get-snip-height x))
-                                                    (λ (x) (get-snip-width x)))
-                                                this-level-snips))])
-                   (let loop ([snips this-level-snips]
-                              [minor-dim (/ (- max-minor this-minor) 2)])
-                     (unless (null? snips)
-                       (let* ([snip (car snips)]
-                              [new-major-coord
-                               (+ major-dim
-                                  (floor
-                                   (- (/ this-major 2) 
-                                      (/ (if vertical? 
-                                             (get-snip-height snip)
-                                             (get-snip-width snip))
-                                         2))))])
-                         (if vertical?
-                             (move-to snip minor-dim new-major-coord)
-                             (move-to snip new-major-coord minor-dim))
-                         (loop (cdr snips)
-                               (+ minor-dim
-                                  (if vertical?
-                                      (get-snip-hspace)
-                                      (get-snip-vspace))
-                                  (if vertical?
-                                      (get-snip-width snip)
-                                      (get-snip-height snip)))))))
-                   (loop (cdr levels)
-                         (+ major-dim 
-                            (if vertical? 
-                                (get-snip-vspace)
-                                (get-snip-hspace))
-                            this-major)))]))))
+          (define levels (sort (hash-map level-ht list) (λ (x y) (<= (car x) (car y)))))
+          (let loop ([levels levels]
+                     [major-dim 0])
+            (cond
+              [(null? levels) (void)]
+              [else
```

Comment:
This is equivalent to `(unless (null? levels) ...)`. Filed jackfirth/resyntax#420.